src/pages/services/index.tsx

```diff
@@ -419,66 +428,61 @@ export default function ServicesPage() {
           <main className='flex-1'>
             <h2 className='sr-only'>Available Services</h2>
             <div
-              className='grid grid-cols-1 md:grid-cols-2 gap-4 md:gap-6'
+              className='grid grid-cols-1 lg:grid-cols-2 xl:grid-cols-3 gap-4 md:gap-6 transition-all duration-300 ease-in-out'
               role='list'
               aria-label='List of government services'
+              tabIndex={-1}
             >
               {paginatedServices.map((service, index) => (
                 <article key={index} className='h-full' role='listitem'>
                   <Card hoverable className='bg-white h-full'>
-                    <CardContent className='p-4 md:p-6'>
-                      <div className='flex items-start justify-between mb-4'>
-                        <div>
-                          <h3 className='text-lg font-semibold text-gray-900'>
-                            {service.service}
-                          </h3>
-                          <div className='mt-2 flex flex-wrap gap-2'>
-                            <Link
-                              to={`/services?category=${service.category.slug}`}
-                              className='inline-block px-2 py-1 text-xs font-medium rounded bg-primary-100 text-primary-800 hover:bg-primary-200 transition-colors'
-                            >
-                              {service.category.name}
-                            </Link>
-                            <Link
-                              to={`/services?category=${service.category.slug}&subcategory=${service.subcategory.slug}`}
-                              className='inline-block px-2 py-1 text-xs font-medium rounded bg-gray-100 text-gray-800 hover:bg-gray-200 transition-colors'
-                            >
-                              {service.subcategory.name}
-                            </Link>
+                    <CardContent className='p-4 md:p-6 h-full'>
+                      <div className='flex flex-col justify-between h-full '>
+                        <div className='flex items-start justify-between mb-4'>
```

Comment:
further style improvement to each service card: add a gap between the service name and the checkmark to prevent the text and the symbol from being too close to each other

```suggestion
                        <div className='flex items-start justify-between gap-2 mb-4'>
```

before:
<img width="600" alt="image" src="https://github.com/user-attachments/assets/8ecbc0a8-1fb3-4871-87a7-3206aff58027" />

after:
<img width="600" alt="image" src="https://github.com/user-attachments/assets/8db376d4-4d95-41a7-adad-3c96d3680504" />